pages/gallery-page.html

```diff
@@ -448,5 +460,6 @@ <h1 class="text-2xl font-semibold mb-6">
       </section>
       <!-- Section 4 ends here -->
     </main>
+    <script src="/js/carousel.js"></script>
```

Comment:
Hello! Is there any reason why the script is not in the head ?:)

Reply:
Hei, Avoid Render-Blocking: JavaScript can be render-blocking, meaning it prevents the browser from rendering other content until the script is downloaded and executed. Placing scripts at the bottom minimizes this issue by allowing the browser to render HTML content first.

Reply:
This is stated in the rules: 

JavaScript 
Coding Standards 
Modules: 
Use files with the ‘.mjs’ extension, which includes ‘import’ and ‘export’ statements to provide better code organization and reusability. 

You might want to consider creating an msj file instead?

![Skjermbilde 2024-04-11 154537](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/146728692/e0dd1113-f650-416c-bee6-64f00f484d52)

Reply:
Our team misunderstood, this code "Gallery-slider.mjs" is belong to one of our team, its not mine. For now you can close this pull request.